README.MD

```diff
@@ -1,3 +1,43 @@
 ## MovieDB - react-native
 
 A (unofficial) port of The MovieDB, written with react-native, typescript and only FunctionComponents!
+
+## Tests
+
+```sh
```

Comment:
You can remove this ```sh```.  Also, would be better say somethink like: 

_To run this projects, just type at your terminal: ```npm i; npm start``` or ```yarn; yarn start```_

Since i dont have implemented relevant tests, i dont see a reason to let this at the MD. :sweat_smile:

Reply:
Btw, have tests. Do you think that can be useful in the future?

Reply:
Sure. Every project should have tests. :tada: